internal/transform/transform.go

```diff
@@ -404,6 +405,17 @@ func ExtractScript(doc *astro.Node, n *astro.Node, opts *TransformOptions, h *ha
 	}
 }
 
+func HintAboutImplicitInlineDirective(n *astro.Node, h *handler.Handler) {
+	if n.Type == astro.ElementNode && n.DataAtom == a.Script && len(n.Attr) > 0 && len(n.Attr) == 1 && n.Attr[0].Key != "src" && !HasInlineDirective(n) {
```

Comment:
Whoops, just realized the logic isn't correct after posting the comment 😅
This should be correct, hopefully
```suggestion
	if n.Type == astro.ElementNode && n.DataAtom == a.Script && len(n.Attr) > 0 && (!HasInlineDirective(n) || len(n.Attr) == 1 && n.Attr[0].Key != "src") {
```

Reply:
Thanks, I'll add a test for external scripts soon as well.

Reply:
Awesome!